image_width: 1280
image_height: 720
camera_name: realsenseD435i
camera_matrix:
  rows: 3
  cols: 3
  data: [906.64027,   0.     , 627.36092,
           0.     , 908.00455, 366.51433,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.130468, -0.256998, -0.002236, -0.002475, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [906.51471,   0.     , 621.24809,   0.     ,
           0.     , 923.29816, 364.86692,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]